SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -386,6 +397,21 @@ TArray<Worker_InterestOverride> USpatialSender::CreateComponentInterest(AActor*
 	return ComponentInterest;
 }
 
+RPCPayload USpatialSender::CreateRPCPayloadFromParams(FPendingRPCParams& RPCParams)
+{
+	check(RPCParams.TargetObject.IsValid());
+	const UObject* Object = RPCParams.TargetObject.Get();
+	const FClassInfo& Info = ClassInfoManager->GetOrCreateClassInfoByClass(Object->GetClass());
+	const FRPCInfo* RPCInfo = Info.RPCInfoMap.Find(RPCParams.Function);
+	FUnrealObjectRef TargetObjectRef(PackageMap->GetUnrealObjectRefFromNetGUID(PackageMap->GetNetGUIDFromObject(Object)));
+	check(TargetObjectRef != FUnrealObjectRef::UNRESOLVED_OBJECT_REF);
+
+	TSet<TWeakObjectPtr<const UObject>> UnresolvedObjects;
```

Comment:
What  happens with these UnresolvedObjects?

Reply:
They are ignored. As I understand, null pointers are used in RPCPayload for them and that is acceptable. @improbable-valentyn please correct me if I've missed something here?

Reply:
That is accurate. We should still comment this behavior and also potentially log a warning if any UnresolvedObjects are actually present. This should be very rare (I believe only possible with singletons in a multiserver scenario).

Reply:
Added warning message.